Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/manager/ReplyManager.java

```diff
@@ -53,7 +53,7 @@ public void putReply(ChanDescriptor chanDescriptor, Reply reply) {
         // Remove files from all other replies because there can only be one picked_file at the same time.
         // Not doing this would be confusing and cause invalid fileNames.
         for (Map.Entry<ChanDescriptor, Reply> entry : drafts.entrySet()) {
-            if (!entry.getKey().equals(chanDescriptor)) {
+            if (entry.getKey() != chanDescriptor) {
```

Comment:
Here should be "equals" not "!=" because this is java, not kotlin.

Reply:
Well ok.

Reply:
Those are different things in Java and Kotlin. In Java "==" is reference comparison which only compares pointers to objects while "equals" compares contents of the objects. In Kotlin "==" does the same "equals" does in Java, and "===" does what "==" does in Java.